java-generator/it/src/it/plugin/gradle/simple/build.gradle

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java library project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/8.1/userguide/building_java_projects.html
+ */
+
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    id 'java-library'
+    id 'io.fabric8.java-generator' version '6.6-SNAPSHOT'
+}
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+}
+
+// Apply a specific Java toolchain to ease working on different environments.
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11)
+    }
+}
+
+javaGen {
+    source = file('artemis-crd.yml')
+    target = file('tmp/generated/')
```

Comment:
Question, is the `target` folder already included in the "Source folders" of Gradle?

Reply:
I am not sure to get the gist here: is your questio about the `target` property key or rather about its value? The reason I am asking: do we want to changethe default _value_ or the test one to something like `build/generated/` or do we want to chenge the _key_ to something in the Gradle semantics? 
If the latter is the case we should ask ourselves whether we agree on having configuration key names to differe from the ones of the Maven plugin.
WDYT?

Reply:
Sorry for the confusion, I'm referring to the "default value".

Reply:
This should be fixed now